### ISSUE TITLE
Suggestion - isthereanydeal.com in ES to only Steam key prices not Uplay key prices

### ISSUE THREAD

**Issue by Grimmyn** [jshackles/Enhanced_Steam#1124]
For example Far Cry® Primal on Steam lists 
http://store.steampowered.com/app/371660/
Current Lowest Price: 23,60€ at GamesPlanet UK (Info)
Historical Lowest Price: 23,60€ at GamesPlanet UK on 2016-08-02 (Info)

But these are all Uplay keys and prices.
I'm not interested in uplay when using Enhanced Steam in Steam. Are you?

Please make it so that Enhanced Steam only shows Current Lowest & Historical Lowest Prices for Steam keys.
Thank you

**Comment by tekanoma** [comment]
I just found this myself but what you're looking for is located in the enhanced steam price options, uncheck "compare all stores" and it will let you select which stores you don't want it to compare

**Comment by Grimmyn** [comment]
That don't work.
It just unlist stores that may or may not offer steam keys.
As my example with Far Cry Primal.
Unlisting GamesPlanet UK is not wanted because they also sell Steam keys.
But not in this case for this game.
The suggestion is that ES looks up isthereanydeal.com and only filter out prices for Steam keys no matter what store.

Thank you for replying to my suggestion.

**Comment by jshackles** [comment]
This currently isn't possible with IsThereAnyDeal's existing APIs.  In over 90% of cases like this, Steam copies of games from other launchers are only available on Steam.  In your example of Far Cry Primal, the only way to get the game on Steam is by purchasing it directly on Steam.  The last Ubisoft game that I'm aware of that launched through Uplay but had Steam keys was Assassin's Creed III - and those Steam keys were only available from Amazon.

As it is, Enhanced Steam will inform you with an "(Activates on Steam)" message in the price history dialog, if the game is known to come with a Steam key.

**Comment by Grimmyn** [comment]
Ah ok.
There is a limit in the API for IsThereAnyDeal.
So where does Enhanced Steam get the information that triggers the "(Activates on Steam)" text?

If "(Activates on Steam)" 100% filters out the Steam keys.
Can that be a feature in Enhanced Steam to only get that?
or that is the problem with the API? The request to IsThereAnyDeal is that ES can't get all data and then filter out only Steam only "(Activates on Steam)" ?

Thank you

**Comment by jshackles** [comment]
Yes that is correct - Enhanced Steam retrieves the requested data from IsThereAnyDeal, and their API shows the lowest price and from which store.  Their API also indicates whether or not that particular deal comes with a Steam key (which triggers the "Activates on Steam" text).  Currently it isn't possible to request only data from stores with Steam keys, although I'll follow up with their developers and see if that is something that could be implemented.

**Comment by Grimmyn** [comment]
If that would be possible it would be fantastic
thank you for looking into this

**Comment by VanityFox** [comment]
So, Enhanced Steam is discontinued. There is now [Augmented Steam](https://github.com/tfedor/AugmentedSteam/). Perhaps your requested feature/bugfix is already implemented there! If not, you could move your issue there.
